pkg/gadgettracermanager/k8s/k8s.go

```diff
@@ -125,31 +125,14 @@ func (k *K8sClient) PodToContainers(pod *v1.Pod) []pb.ContainerDefinition {
 			log.Printf("Skip pod %s/%s: got zero pid", pod.GetNamespace(), pod.GetName())
 			continue
 		}
-		cgroupPathV1, cgroupPathV2, err := containerutils.GetCgroupPaths(pid)
-		if err != nil {
-			log.Printf("Skip pod %s/%s: cannot find cgroup path: %v", pod.GetNamespace(), pod.GetName(), err)
-			continue
-		}
-		cgroupPathV2WithMountpoint, _ := containerutils.CgroupPathV2AddMountpoint(cgroupPathV2)
-		cgroupId, _ := containerutils.GetCgroupID(cgroupPathV2WithMountpoint)
-		mntns, err := containerutils.GetMntNs(pid)
-		if err != nil {
-			log.Printf("Skip pod %s/%s: cannot find mnt namespace: %v", pod.GetNamespace(), pod.GetName(), err)
-			continue
-		}
```

Comment:
Note to myself: Perhaps we could also remove the `pid, err := k.criClient.PidFromContainerId(s.ContainerID)` call above and have a pid enricher. (of course, something to be done in another pr)